src/server/service/kv.rs

```diff
@@ -1450,58 +1451,178 @@ fn future_raw_delete_range<E: Engine, L: LockManager, P: PdClient + 'static>(
     })
 }
 
-// unimplemented
 fn future_ver_get<E: Engine, L: LockManager, P: PdClient + 'static>(
-    _storage: &Storage<E, L, P>,
-    mut _req: VerGetRequest,
+    storage: &Storage<E, L, P>,
+    mut req: VerGetRequest,
 ) -> impl Future<Item = VerGetResponse, Error = Error> {
-    let resp = VerGetResponse::default();
-    future::ok(resp)
+    let mut ver_value = VerValue::default();
+    let version: TimeStamp = req.get_start_version().into();
+    storage
+        .get(
```

Comment:
`storage.get()` is used for txnkv. The logic can not reused by verkv, you should add a new function like `ver_get` and implement the logic of `ver_get()`.(It may need a lot of work)